src/mixins/name-request-mixin.ts

```diff
@@ -51,17 +45,14 @@ export default class NameRequestMixin extends Mixins(DateMixin) {
     }
 
     // If the NR's root state is not APPROVED or CONDITIONAL, it is not consumable.
-    if (nr.state !== NameRequestStates.APPROVED && nr.state !== NameRequestStates.CONDITIONAL) {
+    // EXPIRED or CONSUMED should not return NOT_APPROVED.
+    if (![NameRequestStates.APPROVED, NameRequestStates.CONDITIONAL,
```

Comment:
Can conditional be approved (ie conditionally approved)? is this supposed to be `! NameRequestStates.CONDITIONAL`

Reply:
Yes. `NameRequestStates.CONDITIONAL` means conditionally approved and it can be consumed. Is that what you are asking?

Reply:
For sure, the question was mostly around the ! but you answered that in the other PR. 

But yea, conditional from Name Request is a little tricky because it has an accompanying property which indicates whether the Consent has been received, waived or is still REQUIRED. 

So the state could be CONDITIONAL but without consent received which would result in not quite approved yet. 
Not sure how to handle that here just some food for thought.